src/test/resources/item.feature

```diff
@@ -0,0 +1,22 @@
+Feature: As an API consumer
+  I want to manage the items using the REST API
+
+  Scenario: item using REST
+    When I did a post request "item" with the values:
+      | Content  | new Item|
+    Then The response should contain the data with the string value "Content" and "new Item"
+
+    When I did a put request "item" with the id "9307043" and the values:
```

Comment:
How do you assure that exists that id? What happen if we execute in a different environment(different account)?

I think you need to add a precondition first. or Maybe store the created item in a variable and re-use it for the update scenario.